### DISCUSSION TITLE
STT Transcript missing middle section

### DISCUSSION THREAD

**Discussion by retry-the-user** [deepgram/community#893] [General help]
This is regards to the following STT transaction:
Request ID 4418fe3a-0318-4b90-8593-c1780f234da4    
File size: 728298 
Timestamp: 2024-08-12 14:10:26 (UTC)
Model: nova-2-phonecall
Parameters: filler_words=true&smart_format=true

The result came back like so: "Hello. Your verified now has been trying to contact you about your Google business listing. Our records show your listing is not properly verified or optimized with Google. This may cause *MISSING CONTENT HERE* the status of your listing.  This will only take a few minutes. Press 9 or call 6 the status of your listing. This will only take a few minutes. Press 9 or call 619-935-1051 to opt out."

The missing content in the middle of that is "customers searching for your business to not be able to find your business online. Press 1 to speak with a listing expert so we can quickly check".

Subsequent calls for STT result in a complete transcript, but this speaks to the reliability of the DG service.  Missing text is not something we can easily detect in order to retry, nor should we have to when we're paying for it.  I would like to know what happened.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@retry-the-user  could you share your audio file so we can try to reproduce?
